tools/windows-wsl-local-node-config/README.md

```diff
@@ -0,0 +1,104 @@
+Windows WSL Local Node Config Creation Tool
+========================
+
+**Run a single command to create node configuration files to enable running of a local network for ot-node development.**
+
+This tool will generate configuration files for mulitple OriginTrail nodes, allowing you to run a multi node network locally.
+
+**Note:** This tool is intended for creating development nodes only, running on **Windows Subsystem for Linux (WSL).** For MacOS and Linux, see [MacOS & Linux local network setup]([https://docs.origintrail.io/dkg-v6-upcoming-version/setting-up-your-development-environment#running-multiple-nodes-in-local).
+
+## Prerequisites
+
+* A Windows Subsystem for Linux (WSL) environment running Ubuntu 20.04, and configured for OT-Node development.
+  * See document: **How to setup a Windows development environment** (currently in review)
+
+Usage
+=====
+
+## 1. Specify your Node Network ID 
+
+Update the temaplte file `dh_node_config_template` to include your testnet node network id in the bootstrap config: 
+
+```json
+{
+    "bootstrap": [
+      "/ip4/0.0.0.0/tcp/9000/p2p/<your-network-id>"
+    ]
+}
+```
+
+Replace `<your-network-id>` with your network id. This can be found by starting your node and checking the logs:
```

Comment:
is it a prerequisite to be running a testnet node already ? wouldn't generating a new network id from private and public keys make it easier ?

Reply:
Yeah that's probably a better approach. I'll update the script to generate from the keys.

Reply:
In `external/libp2p-service.js` : 

`if (!configFile.network.privateKey) {
                    const id = await PeerId.create({bits: 1024, keyType: 'RSA'})
                    configFile.network.privateKey = id.toJSON().privKey;
                    if(process.env.NODE_ENV !== 'development') {
                        fs.writeFileSync(this.config.configFilename, JSON.stringify(configFile, null, 2));
                    }
                }`
                
So there is really no need to provide network private key. If it's not present, it will be automatically generated. On creation, the only things you need to provide are blockchain public and private keys. Does this answer your question ?

Reply:
And sorry, I reread my previous comment and I certainly implied network id was created from blockchain keys, which isn't correct.

Reply:
@zeroxbt yes thanks. I think we're on the same page now. I'll update the script and documentation :)

Reply:
Thanks and great PR

Reply:
@zeroxbt I've updated script to generate the network private key and node id by default and included an optional step in the documentation for using an existing private key if wanting to.